src/sets.jl

```diff
@@ -344,3 +344,30 @@ end
 function moi_set(set::Semiinteger{T}) where {T}
     return MOI.Semiinteger{T}(set.lower, set.upper)
 end
+
+"""
+    Parameter(value)
+
+A short-cut for the [`MOI.Parameter`](@ref) set.
+
+## Example
+
+```jldoctest
+julia> model = Model();
+
+julia> @variable(model, x in Parameter(2.0))
```

Comment:
The reader might think it's needed to wrote `2.0` while actually `2` is sufficient since we convert automatically as he will see in the `print` below